hardhat.config.js

```diff
@@ -37,7 +38,7 @@ module.exports = {
         version: '0.5.12'
       },
       {
-        version: '0.6.12',
+        version: '0.7.3',
```

Comment:
Any reason for 0.7.3?

ToB recommends 0.7.5-0.7.6
https://github.com/crytic/slither/wiki/Detector-Documentation#incorrect-versions-of-solidity

Reply:
Since the Matic gateway contracts inherit from `FxPortal`, we have to use `0.7.3` for them at least. 

If we think this version is buggy, I can upgrade the rest back to `0.7.6`, and just use `0.7.3` for the gateways.

Reply:
Probably easiest to stay simple with the same 0.7.3 version across.